00-average/Average.java

```diff
@@ -0,0 +1,25 @@
+import java.util.Scanner;
+public class  Average{
+  public static void main(String[] args){
+    System.out.print("Number of grades: ");
+    float num = Input();
+
+    System.out.println("Average: " + Calculation(num)/num);
```

Comment:
for scores that would lead to an average that's a decimal (like 99 & 100 would yield an average of 99.5) does your program output that or just the whole #?